<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warehouse Workers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="videoTab">
       <attribute name="title">
        <string>Warehouse Video Feed</string>
       </attribute>
       <widget class="QLabel" name="videoViewLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>640</width>
          <height>480</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>960</width>
          <height>720</height>
         </size>
        </property>
        <property name="text">
         <string>Video Capture View</string>
        </property>
       </widget>
       <widget class="QPushButton" name="exitButton">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>470</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
       <widget class="QLabel" name="roverPositionLabel">
        <property name="geometry">
         <rect>
          <x>701</x>
          <y>40</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Position:</string>
        </property>
       </widget>
       <widget class="QLabel" name="roverPositionValueLabel">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>40</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>position</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cleanUpButton">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>215</y>
          <width>101</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Clean Up</string>
        </property>
       </widget>
       <widget class="QLabel" name="roverHeadingValueLabel">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>60</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>heading degrees</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="pathSelectSpinBox">
        <property name="geometry">
         <rect>
          <x>829</x>
          <y>214</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
       <widget class="QLabel" name="roverStatusLabel">
        <property name="geometry">
         <rect>
          <x>701</x>
          <y>10</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Rover Status:</string>
        </property>
       </widget>
       <widget class="QLabel" name="roverHeadingLabel">
        <property name="geometry">
         <rect>
          <x>701</x>
          <y>60</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Heading:</string>
        </property>
       </widget>
       <widget class="QLabel" name="pathViewLabel">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>200</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Path View:</string>
        </property>
       </widget>
       <widget class="QLabel" name="pathSelectionLabel">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>222</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Path Selection</string>
        </property>
       </widget>
       <widget class="QLabel" name="cameraFeedControlLabel">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>130</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Camera Feed Control:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshViewButton">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>160</y>
          <width>111</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Refresh View</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="toggleHomePtsCheckbox">
        <property name="geometry">
         <rect>
          <x>695</x>
          <y>296</y>
          <width>171</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Home Points</string>
        </property>
       </widget>
       <widget class="QPushButton" name="goHomeButton">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>83</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Go Home</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="toggleHomePathCheckbox">
        <property name="geometry">
         <rect>
          <x>695</x>
          <y>271</y>
          <width>180</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Path To Home</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="showObjectDetectionCheckbox">
        <property name="geometry">
         <rect>
          <x>818</x>
          <y>163</y>
          <width>190</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Object Detection</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendNextButton">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Send Next</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopSendingButton">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>70</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="showObjectPathCheckbox">
        <property name="geometry">
         <rect>
          <x>695</x>
          <y>250</y>
          <width>190</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Path To Object</string>
        </property>
       </widget>
       <widget class="QPushButton" name="adjustButton">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>90</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Adjust</string>
        </property>
       </widget>
       <widget class="QComboBox" name="dest_box">
        <property name="geometry">
         <rect>
          <x>787</x>
          <y>389</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_candy">
        <property name="geometry">
         <rect>
          <x>698</x>
          <y>362</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Candy</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_dest">
        <property name="geometry">
         <rect>
          <x>698</x>
          <y>386</y>
          <width>91</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>Destination</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_deliver">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>373</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Deliver</string>
        </property>
       </widget>
       <widget class="QComboBox" name="candy_box">
        <property name="geometry">
         <rect>
          <x>787</x>
          <y>356</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="pathViewLabel_2">
        <property name="geometry">
         <rect>
          <x>699</x>
          <y>331</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Delivery:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="movementTab">
       <attribute name="title">
        <string>Movement</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>290</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QPushButton" name="bttn_turnRight">
           <property name="text">
            <string>Turn Right</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="bttn_turnLeft">
           <property name="text">
            <string>Turn Left</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="bttn_forward">
           <property name="text">
            <string>Forward</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="bttn_backward">
           <property name="text">
            <string>Backward</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="bttn_stop">
           <property name="text">
            <string>STOP</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="txt_distance"/>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="bttn_testOff">
           <property name="text">
            <string>Testing Off</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="bttn_testOn">
           <property name="text">
            <string>Testing On</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_encoders">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>340</y>
          <width>271</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="testTab">
       <attribute name="title">
        <string>Sensors</string>
       </attribute>
       <widget class="QLabel" name="proximity_lb">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Proximity:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cm_button">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>80</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>cm</string>
        </property>
       </widget>
       <widget class="QLabel" name="line_value">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>80</y>
          <width>121</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>00000000</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clear_debug">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>60</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear Debug</string>
        </property>
       </widget>
       <widget class="QLabel" name="distance">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="proximity_alert">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>121</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Safe</string>
        </property>
       </widget>
       <widget class="QLabel" name="status_lb">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
       <widget class="QLabel" name="curr_line_lb">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>30</y>
          <width>251</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Current Line Status:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="wait_button">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>90</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Wait</string>
        </property>
       </widget>
       <widget class="QPushButton" name="debug_line">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>80</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Debug Line</string>
        </property>
       </widget>
       <widget class="QPushButton" name="debug_adc">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>30</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Debug ADC</string>
        </property>
       </widget>
       <widget class="QLabel" name="curr_dist_lb">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>221</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Current Distance:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="raw_button">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>raw</string>
        </property>
       </widget>
       <widget class="QLabel" name="status">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>70</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Running</string>
        </property>
       </widget>
       <widget class="QPushButton" name="push_button">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Push</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToggleFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutUs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionToggleFullscreen">
   <property name="text">
    <string>Toggle Full-screen</string>
   </property>
  </action>
  <action name="actionAboutUs">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
